browser/dasher/messageStore.js

```diff
@@ -27,8 +28,12 @@ export default class MessageStore {
 
     viewMessageStore(){
       this._load_from_browser().catch(() => {}).then(messages => {
-        //TODO - present these message in another place other than the console
-        console.log(messages);
+        //TODO - present these messages in a usable format
+        let output = [];
```

Comment:
Maybe use Array flat() to do this instead.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat